README.md

```diff
@@ -1,50 +1,38 @@
 # n-ui
 
-## What is this?
+An experiment bunching some of Next's client-side code into a single repository. [Motivation](explainer.md)
 
-An experiment bunching some of Next's client-side code into a single repository.
+## API
 
-## Motivations
+### Bootstrapping your app
 
-### Ease integration
+Sass and JS follow similar apis. Both expect a map of feature names (see main.js or configure.scss for feature lists), and also recognise 2 presets:
+- complete: all the bits and pieces you need to style and add interactivity to a typical page on next, complete with myft, alerts, promos, ads, tracking etc.
+- discrete: provides styles and js for just the header, footer and essential pieces of functionality such as tracking. For use on pages where we want no distractions e.g. errors, login and other forms
 
-- 25+ Origami components
-- 15+ n- UI components
-- Far fewer places update
-- Fewer conflicts!
+Examples:
 
-### Increase oversight
+```Javascript
+import { bootstrap } from n-ui;
 
-- Easy to find precedent
-- More people working in a shared space
-- A single place to report issues
+bootstrap({
+	preset: 'discrete',
+	welcomeMessage: true
+}, ({flags}) => {
+	if (flags.get('konami')) {
+		easterEgg.init();
+	}
+})
+```
 
-### Reduce duplication
+```Sass
+@import 'n-ui/configure';
+@include nUiConfigure((
```

Comment:
Should this be a function rather than a mixin as it has no output? Maybe print the config here?